src/Components/Footer/Footer.tsx

```diff
@@ -0,0 +1,13 @@
+import './Footer.scss'
+
+type FooterProps = {
+    children: React.ReactNode
+}
+
+export default function Footer(props : FooterProps) {
```

Comment:
Please don't use default export. It could be uncomfortable because
1. default export allows you to rename components as developers want and each developer could have personal vision.
2. it could be problem to use combine export